### ISSUE TITLE
trying to paste large geometries result in error

### ISSUE THREAD

**Issue by chrisgraf** [nextgis/qgis_geometry_paster#3]
An error has occurred while executing Python code: 

_csv.Error: field larger than field limit (131072) 
Traceback (most recent call last):
  File "/home/graf/.local/share/QGIS/QGIS3/profiles/default/python/plugins/geometry_paster/plugin.py", line 242, in _checkPasteAvalability
    if len(self._tryGetFeaturesGeomsFromClipBoard()) == 0:
  File "/home/graf/.local/share/QGIS/QGIS3/profiles/default/python/plugins/geometry_paster/plugin.py", line 203, in _tryGetFeaturesGeomsFromClipBoard
    for row in reader:
  File "/usr/lib/python3.6/csv.py", line 112, in __next__
    row = next(self.reader)
_csv.Error: field larger than field limit (131072)

Python version: 3.6.8 (default, Apr  9 2019, 04:59:38) [GCC 8.3.0] 
QGIS version: 3.8.1-Zanzibar Zanzibar, dcd95cc648 

Python Path:
/usr/share/qgis/python
/home/graf/.local/share/QGIS/QGIS3/profiles/default/python
/home/graf/.local/share/QGIS/QGIS3/profiles/default/python/plugins
/usr/share/qgis/python/plugins
/usr/lib/python36.zip
/usr/lib/python3.6
/usr/lib/python3.6/lib-dynload
/usr/local/lib/python3.6/dist-packages
/usr/lib/python3/dist-packages
/home/graf/.local/share/QGIS/QGIS3/profiles/default/python
/media/ilen-q/Projekte/Retour/GISdaten
.

**Comment by chrisgraf** [comment]
adding the line from the following thread seems to fix the issue:
https://stackoverflow.com/questions/15063936/csv-error-field-larger-than-field-limit-131072

**Comment by tomass** [comment]
To be more specific, in order for fix to work on both linux and windows, you need to add to _plugin.py_

`import ctypes as ct`

and then add a line before reader = csv.DictReader (around line ~197):

```
if sys.version_info[0] == 2:
    clipboard_text = clipboard_text.encode('utf-8')

csv.field_size_limit(int(ct.c_ulong(-1).value // 2)) # <---- THIS IS THE FIX LINE
reader = csv.DictReader(
    StringIO(clipboard_text),
    delimiter='\t'
)

```

**Comment by simgislab** [comment]
hi there, can you send us a PR?

**Comment by simgislab** [comment]
thanks, just published a new version of the plugin (0.2) with this fix

hope it helps
